elements/nuxeo-document-history/nuxeo-document-history.html

```diff
@@ -35,31 +37,62 @@
 <dom-module id="nuxeo-document-history">
   <template>
     <style>
-      #heading {
+      .row-container {
         @apply --layout-horizontal;
-        @apply --layout-end-justified;
+        @apply --layout-wrap;
       }
 
-      #table {
-        height: calc(100vh - 370px);
+      .row-container * {
+        flex: 1 0 0;
+        margin-right: 8px;
       }
 
-      nuxeo-date-picker {
-        padding: 0 16px;
+      #table {
+        height: calc(100vh - 370px);
```

Comment:
We do get a double scrollbar (if we get more content on the table), which I don't think we want. Can you reproduce? Could you please try with:
```suggestion
        height: calc(100vh - 420px);
```

Reply:
Thanks, I increase the part of fixed height. Just a a quick question How did you check this part I mean you retrieved my branch and test or do you have another way to check it

Reply:
I made a checkout, because I was curious about the min/max of the date picker (I don't think I have ever used it :thinking:), a it works quite well! :P

Reply:
Thanks ;), yeah  for `min` and `max` it works really good :)